packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -119,6 +114,30 @@ const VirtualListBaseFactory = (type) => {
 			 */
 			initUiChildRef: PropTypes.func,
 
+			/**
+			 * The Function that returns `true` if the item at the index is disabled
+			 *
+			 * Usage:
+			 * ```
+			 * isDisabledItem = (index) => (this.items[index].disabled)
+			 * render = () => {
+			 * 	return (
+			 * 		<VirtualList
+			 * 			dataSize={this.items.length}
+			 * 			isDisabledItem={isDisabledItem}
+			 * 			itemRenderer={this.renderItem}
+			 * 			itemSize={this.itemSize}
+			 * 		/>
+			 * 	);
+			 * }
+			 * ```
+			 *
+			 * @type {Function}
+			 * @param {Number} index
+			 * @public
+			 */
+			isDisabledItem: PropTypes.func,
```

Comment:
This property is not required yet there is no default or fallback specified.  Would it make sense to assume no items are disabled if this is not supplied and document it that way?